packages/notification-center/src/utils/date.ts

```diff
@@ -0,0 +1,39 @@
+const SECOND = 1000;
+const MINUTE = SECOND * 60;
+const HOUR = MINUTE * 60;
+const DAY = HOUR * 24;
+const YEAR = DAY * 365;
+
+const UNITS = {
+  year: YEAR,
+  month: YEAR / 12,
+  day: DAY,
+  hour: HOUR,
+  minute: MINUTE,
+  second: SECOND,
+};
+
+const DEFAULT_LOCALE = 'en';
+
+export function formatRelativeTime({
```

Comment:
the similar function that `date-fns` provides `formatDistanceToNow`, but this one uses the `Intl.RelativeTimeFormat`